plugins/inputs/t128_tank/t128_tank.go

```diff
@@ -33,6 +34,12 @@ var sampleConfig = `
 
 ## Server Address to get tank data from.
 # server_address = "127.0.0.1"
+
+## From specifies the first message we are interested in.
+## If from is "beginning" or "start", it will start consuming from the 
+## first available message in the selected topic. If it is "eof" or "end", 
+## it will tail the topic for newly produced messages.
+# from = "end"
```

Comment:
add a validation to make sure it is one of the 4 values so that users aren't shooting themselves in the foot?

Reply:
Don't support all these values. `start` and `end` gives us all the functionality we need.